spec/database_cleaner/sequel/deletion_spec.rb

```diff
@@ -1,7 +1,8 @@
-require 'spec_helper'
 require 'database_cleaner/sequel/deletion'
 require 'database_cleaner/shared_strategy'
 require 'sequel'
+require 'support/sequel/sequel_setup'
+# XXX: use ActiveRecord's db_config (`db/config.yml`) for CI/dev convenience
```

Comment:
@botandrose Interesting. Is it not using it now?

Reply:
Honestly, IDK. I just copied it over along with the line following it from `spec/database_cleaner/sequel/truncation_spec.rb`. I think its intended as an FYI, but maybe its an outdated TODO, and we can just ditch it. I'll git blame it, and find out.

Reply:
Ok, looks like its an advisory comment. I think its just saying something like "despite the fact that we're testing the Sequel ORM, let's just cheat and reuse the ActiveRecord connection .yml file, because its basically free."